test_balboa_spa_component.yaml

```diff
@@ -14,29 +14,45 @@ balboa_spa:
 switch:
   - platform: balboa_spa
     balboa_spa_id: test_spa
-    jet1:
-      name: Jet1
-    jet2:
-      name: Jet2
-    jet3:
-      name: Jet3
-    jet4:
-      name: Jet4
+    jet1: # Test if basic switch works
```

Comment:
For the test component used for build validation, so please leave these as-is.  

I appreciate most users will not have both, but it allows us to use Github Actions to make sure all of these compile correctly.